gr-uhd/swig/uhd_swig.i

```diff
@@ -52,6 +58,15 @@
 #include <gnuradio/uhd/usrp_source.h>
 #include <gnuradio/uhd/usrp_sink.h>
 #include <gnuradio/uhd/amsg_source.h>
+#include <gnuradio/uhd/rfnoc_block.h>
```

Comment:
The `#ifdef` around these might need in its own `%{` scope ... have to experiment to see if `#ifdef` works inside it or not.